README

```diff
@@ -1,72 +0,0 @@
-HAWK version 0.9.8
```

Comment:
This file is redundant with README.md, and some parts are out of date. Better to just drop it.